src/sentry/interfaces/stacktrace.py

```diff
@@ -183,15 +187,20 @@ def remove_filename_outliers(filename, platform=None):
     # contain things like /Users/foo/Dropbox/...
     if platform == 'cocoa':
         return posixpath.basename(filename)
+    elif platform == 'java':
+        filename = _java_assist_enhancer_re.sub(r'\1<auto>', filename)
```

Comment:
you sure your test is actually hitting this path? codecov says not

Reply:
Oh yeah, I think having a module set makes the module "win." Need another case.

Reply:
Fixed.